searchlib/src/vespa/searchlib/queryeval/nearest_neighbor_iterator.cpp

```diff
@@ -112,11 +116,16 @@ NearestNeighborIterator::create(
         const vespalib::tensor::DenseTensorView &queryTensor,
         const search::tensor::DenseTensorAttribute &tensorAttribute,
         NearestNeighborDistanceHeap &distanceHeap,
+        const search::BitVector *bit_vector,
         const search::tensor::DistanceFunction *dist_fun)
 
 {
-    Params params(tfmd, queryTensor, tensorAttribute, distanceHeap, dist_fun);
-    return resolve_strict_LCT_RCT(strict, params);
+    Params params(tfmd, queryTensor, tensorAttribute, distanceHeap, bit_vector, dist_fun);
```

Comment:
Consider renaming _bit_vector_ to _filter_ to match how it is named in hnsw index.